modules/apps/frontend-js/frontend-js-alert-support-web/npmscripts.config.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+module.exports = {
+	federation: {
+		bridges: [],
```

Comment:
Do we need to specify an empty array? Can't we simply avoid this?

Reply:
Also, I don't see a `main` entry in the `package.json` in this module. Is it not needed to work with `federation`?

Reply:
We can change it to `true` instead of an array, but it is necessary, because it tells `npm-scripts` if the `main` entry point of the package must be bridged or not. It is "simply" activating the bridges but declaring that we don't have any external package to bridge.

Regarding the `main` entry, for now we only bridge that because we have the convention that everything is exported through it, but it is possible that in the future we may need to bridge named modules too (if we still make use of them anywhere).

Reply:
> We can change it to true instead of an array, but it is necessary, because it tells npm-scripts if the main entry point of the package must be bridged or not. It is "simply" activating the bridges but declaring that we don't have any external package to bridge.

So, this is to create a Loader2Webpack bridge, but we're changing the way this loads to use the new LoaderSupport, don't we? As such... why would we want to generate the bridges? No one was consuming this, specially since it had no `main` entry point, so it should be relatively safe to avoid doing unnecessary work.

Reply:
There's no `main` entry because the file is named `index.js` so, unless we know for sure that nobody is using it, we cannot be sure by looking at the code...

Reply:
Let's say we're sure :D